Tensile/Common.py

```diff
@@ -281,7 +281,18 @@
     "WaveSeparateGlobalReadB":    [ 0, 1 ],
 
     "PrefetchGlobalRead":         [ False, True ], # prefetch / double-buffer reads from global memory -> vgprs -> lds. Requires 2X LDS space, and VGPRs for buffering data on way into LDS
-    "PrefetchLocalRead":          [ 0,1,2,3], # prefetch / double-buffer reads from lds (or 2 for triple-buffer, 3 for quad-buffer).  Increases size of ValuA/ValuB registers.
+    "PrefetchLocalRead":          list(range(128+1)), # prefetch / double-buffer reads from lds (or 2 for triple-buffer, 3 for quad-buffer).  Increases size of ValuA/ValuB registers.
```

Comment:
in my calculation, bf16 with MT64x64x256_MI32x32x4x1 will have 64 iteration,
so that we can use PLR=1,3...64,65,67...
I add more comment about PLR setting